src/net.h

```diff
@@ -769,8 +769,6 @@ class CNode
     static uint64_t GetMaxOutboundTimeLeftInCycle();
 };
 
-
-
```

Comment:
@dragongem45 Are you still working on this?

Please remove this diff and squash your commits.